src/pages/register-a-beacon/about-beacon-owner.tsx

```diff
@@ -18,7 +18,7 @@ import { Validators } from "../../lib/form/validators";
 import { CacheEntry } from "../../lib/formCache";
 import { FormPageProps, handlePageRequest } from "../../lib/handlePageRequest";
 
-const getFormManager = ({
+const defineFormRules = ({
```

Comment:
@zackads See comment above regarding naming of this function in the `handlePageRequest` file.  I am not going to add this comment on all files but highlighting it on here so you are aware that I would update this to reflect whatever name is given to the typed function in that file